Make.h

```diff
@@ -0,0 +1,19 @@
+//
+//  Make.h
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/8/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import <Mantle/Mantle.h>
+
+@interface Make : MTLModel <MTLJSONSerializing>
+@property (nonatomic,retain) NSNumber* makeID;
+@property (nonatomic,retain) NSString* nameEn;
+@property (nonatomic,retain) NSString* nameAr;
+@property (nonatomic,retain) NSNumber* modelsCount;
+@property (nonatomic,retain) NSNumber* enginesCount;
+@property (nonatomic,retain) NSString* imageSmallURL;
+@property (nonatomic,retain) NSString* imageLargeURL;
+@end
```

Comment:
The task requirements needs make ID, arabic name, english name and the image why you get models and engine count ?